test/functional/Java8andUp/src/org/openj9/test/java/security/DigestLoop.java

```diff
@@ -75,6 +93,12 @@ protected long loop(long numIterations) throws InterruptedException {
         Thread currentThread = Thread.currentThread();
         byte[] result = null;
         Random r2 = new Random(currentThread.getId());
+        MessageDigest md = null;
+        try {
+            md = MessageDigest.getInstance(alg);
+        } catch (NoSuchAlgorithmException e) {
+            logger.debug(e);
```

Comment:
The test should fail here rather than throw a `NullPointerException` below.

Reply:
Fixed this and added the final.